src/components/Buttons/BackButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, ReactElement } from "react";
+import cn from "classnames";
+import Link from "../Link";
+import "./BackButton.css";
+
+type Props = {
+  className: string;
+  to: string;
+  size: number;
```

Comment:
**suggestion** I guess `size` and `className` properties can be optional.

Reply:
ok, I fix that